components/Form/Password.js

```diff
@@ -53,6 +53,9 @@ class Password extends React.Component {
     }
     if (event.target.value === this.state.passwordTwo) {
       this.setState({ displayWarningMatch: false });
+      if (event.target.value === "") {
```

Comment:
I don't understand this change. I think the 'Create' button should be disabled whenever some mandatory field is missing from the modal.
That said, having the password erased should always disable the Create button (except if the password is not mandatory field)

Reply:
I'm on the fence about that. My general inclination is to provide the user the freedom to do what they want, and not place restrictions on user actions based on our assumptions about how things should/shouldn't work. I would suggest we only require fields that would result in an api failure. And in cases like this where we know the primary use case for defining a user account would fail, we can always inform them with an inline warning about the consequences. But I don't think we should prevent them from adding information to a user account and saving those changes. 

cc'ing @larskarlitski and @andreasn for their input on this

For this case, should we display an inline warning? (and would that be a separate PR?)

Reply:
I don't really have strong opinion on this. We can introduce a warning in a seperate PR definitely.

Reply:
I don't feel strongly about whether we're showing a warning or disallowing it completely.